### PR TITLE
fix broken archival Scala Days links

### PR DESCRIPTION
**Pull request by SethTisue**
as noticed by failing CI

### CODE REVIEW

File: _events/2014-06-16-scaladays.md [hunk 1]
```diff
@@ -6,5 +6,5 @@ location: Berlin, Germany
 description: ""
 start: 16 June 2014
 end: 18 June 2014
-link-out: https://scaladays.org/assets/archive/berlin2014/index.html
+link-out: https://scaladays.org/
```
Comment (eed3si9n) [inline _events/2014-06-16-scaladays.md:9]: Did we lose historical Scala Days sites? If so could we bring them back?

Comment (SethTisue) [reply _events/2014-06-16-scaladays.md:9]: hmm, not sure who to ask, maybe @MaureenElsberry ?

I think we should merge the PR regardless to fix CI, but I'd be happy to revert it again later if the archives were resurrected.

**Comment by MaureenElsberry** [comment]
@SethTisue @eed3si9n - we will be bringing them back, but we only had time to create a splash page when Lightbend transferred the domain to us. Israel is working on additions soon. I think it's fine to archive it for now and I can let you know when the pages are back up!

**Review by eed3si9n — APPROVED** [review]

**Comment by SethTisue** [comment]
thanks Maureen!

**Pull request merged**